### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by cohnjesse**
## Description
[Original Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/42343)

In order to better document our applications we need to create detailed README.md files that include everything an engineer needs to get a healthy understanding of the application from a UX and front end code perspective as well as any mocking they need to run the app. This PR adds that README.md file.

## Testing done
NA

## Screenshots
NA

## Acceptance criteria
- [x] The COE status app now has a detailed README.md file

### CODE REVIEW

**Review by scottymeyers — APPROVED** [review]

**Review by Mottie — APPROVED** [review]

File: src/applications/lgy/coe/status/README.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+## How the app works for the user
+When a Veteran first lands on the page we make the initial call to the LGY service to get the status of the COE for the Veteran. We then show content relevent to that Veteran based on the status that comes back from LGY, so for example if the Veteran is automatically approved for a COE we show them a message that says that they are automatically approved as well as a link to download the COE as a PDF.
+
+## The front end code
+The front end for the COE status app is broken up into `containers` and `components` with the main entry point of the app being `app-entry.jsx`. Inside the `containers` directory there is a file called `App.js` which housese the main logic for the front end, everything else are components rendered inside this main `App.js` file. We use Redux to make the call to the LGY endpoints for the status of the COE, those are one level out in a `/shared` folder since they are used in this app and in the form app.
```
Comment (jerekshoe) [inline src/applications/lgy/coe/status/README.md:16]: which **_houses_** the main logic

File: src/applications/lgy/coe/status/README.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+Inside `App.js` we import the Redux actions and based on the data returned by the LGY service called in those Redux actions we enter into a block of logic. That logic first tests if the call to LGY is pending and if it is it returns a loading indicator. Once the call to LGY is successful we enter into a switch statement based on the data returned by the LGY service. 
+
+
+### Statuses
```
Comment (jerekshoe) [inline src/applications/lgy/coe/status/README.md:21]: Do we want to include this here? This level of detail seems a bit much for frontend considerations

File: src/applications/lgy/coe/status/README.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
+App Name: `Certificate of Eligibility (COE) Status App`
+Active engineers: 
+Form ID (if different from app name, NA otherwise): `NA`
+URL: `{root url}/housing-assistance/home-loans/check-coe-status/your-coe`
+
+
+# Background
+When a Veteran wants to get a VA backed home loan the VA can provide them with something called a "Certificate of Eligibility" (COE). This certificate does not approve them for a specific dollar amount but rather lets the Veteran's chosen lender know that they are in fact able to get a VA backed home loan.
```
Comment (jerekshoe) [inline src/applications/lgy/coe/status/README.md:8]: VA backed should be `VA-backed`

File: src/applications/lgy/coe/status/README.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
+App Name: `Certificate of Eligibility (COE) Status App`
+Active engineers: 
+Form ID (if different from app name, NA otherwise): `NA`
+URL: `{root url}/housing-assistance/home-loans/check-coe-status/your-coe`
```
[the remark this answers is not in the document]
Comment (cohnjesse) [reply src/applications/lgy/coe/status/README.md:4]: Im not sure it does, I was going to do separate one for the form, there is more to the form and the stuff it does use from the shared folder is used a bit differently

File: src/applications/lgy/coe/status/README.md [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+
+
+# Background
+When a Veteran wants to get a VA backed home loan the VA can provide them with something called a "Certificate of Eligibility" (COE). This certificate does not approve them for a specific dollar amount but rather lets the Veteran's chosen lender know that they are in fact able to get a VA backed home loan.
+
+When a veteran wants to recieve a COE they can come to this page and we can make a call to the upstream data service LGY and they can send us a determination of if the Veteran is automatically approved for a COE. If they are automatically approved then we can provide them a link to download that COE. Alternatively if the Veteran applied for a COE then the call we make the LGY serivice will return to us the status of that COE application that the Veteran has in progress.
```
[the remark this answers is not in the document]
Comment (jerekshoe) [reply src/applications/lgy/coe/status/README.md:10]: we make the LGY **_service_**..

File: src/applications/lgy/coe/status/README.md (hunk 2 above)
Comment (cohnjesse) [reply src/applications/lgy/coe/status/README.md:21]: I removed the portion of back end code but I think the rest is still good, what do you think?

**Comment by jerekshoe** [comment]
One thing that might be useful for the other readme would be to have a section for users that can be used with the application. Other than that, this looks good to me!

**Comment by theodur** [comment]
Merging the latest changes in `main` should get CI passing for this PR. There was a bug for PRs with only Markdown changes that's been fixed

**Pull request closed without merging**